### PR TITLE
Displayed Confirm and Cancel buttons on editing node properties

### PR DESCRIPTION
**Pull request by nehashende**

### CODE REVIEW

File: src/browser/components/buttons/ConfirmationButton.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class ConfirmationButton extends Component {
   }
 
   render () {
-    if (this.state.requested) {
+    if (this.state.requested || this.props.requested) {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/components/buttons/ConfirmationButton.jsx]: revert the change and if required come up with your own component which has confirm and cancel button.

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 2]
```diff
@@ -92,13 +92,15 @@ EntitySection.propTypes = {
 }
 
 /**
- * Properties section
+ * Component to display the properties of selected node
  * @param {*} props
  */
 
-export const PropertiesSection = props => {
+const DisplayProperties = props => {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: extract in separate module

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

File: src/browser/components/buttons/ConfirmationButton.jsx (hunk 1 above)
Comment (nehashende) [reply src/browser/components/buttons/ConfirmationButton.jsx]: Added a new component

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx (hunk 2 above)
Comment (nehashende) [reply src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: Done with changes
